processors/games/HungerGames.js

```diff
@@ -173,6 +178,8 @@ module.exports = ({
         teams: getKitStat(/^wins_teams_/),
       },
       kills: getKitStat(/^kills_/),
+      kd: getRatio(getKitStat(/^kills_/),getKitStat(/^deaths_/)),
```

Comment:
Just a minor thing, `k_d` and `w_l` are used in other modes so using those would be good for concistency.